SourceCode2Pdf/PythonScriptsExecution/PythonScriptsExecution.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using IronPython.Hosting;
+using Microsoft.Scripting.Hosting;
+
+namespace SourceCode2Pdf.PythonScriptsExecution
+{
+    public class PythonScriptsExecution<T>
+    {
+        public T Execute(string scriptFile)
+        {
+            ScriptEngine pythonScript = Python.CreateEngine();
+            ScriptScope scope = pythonScript.CreateScope();
```

Comment:
add using if scope is IDisposable

Reply:
ScriptScope class doesnt implement IDispose interface